inst/templates/ad_adrs_basic.R

```diff
@@ -85,12 +85,24 @@ adrs <- adrs %>%
 # parameter derivations - here only valid assessments and those occurring on or
 # after randomization date, if there is more than one assessment per date the
 # worst one is flagged
+worst_resp <- function(arg) {
+  case_when(
+    arg == "CR" ~ 1,
+    arg == "PR" ~ 2,
+    arg == "SD" ~ 3,
+    arg == "NON-CR/NON-PD" ~ 4,
+    arg == "NE" ~ 5,
```

Comment:
should not we have NE with a value of 1?

Reply:
Yes, that's better. Then we are consistent with the iRECIST vignette..

I have updated it.